spec/e2e/unpersisted_association_spec.rb

```diff
@@ -237,7 +237,7 @@ def save_and_expect_rel!
           chris = Student.create(name: 'Chris', favorite_class: math)
           expect(chris.errors).to be_empty
 
-          lessons = chris.query_as(:c).match('c-[:FAVORITE_CLASS]->(l:Lesson)').pluck('l.uuid')
+          lessons = chris.query_as(:c).match('c-[:FAVORITE_CLASS]->(l:Lesson)').pluck(l: Lesson.id_property_name)
```

Comment:
Here `pluck(l: :id)` is not working as match returns directly object of neo4j-core so no way to intercept and not enough information in neo4j-core to implement. I think that all those chaining methods should return some proxy in neo4j rather neo4j-core. The same problem occurs with where clauses. This is not specific to neo_id.